tests/lib/failover/inject_rclocalcrash.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Can we please inline this logic until we have a second user that would justify putting it in the library?  It's very specific to the test, and uses variable names which the call site must know about. Per our call yesterday, it's better to avoid that so we don't need to dig into every file for finding out what actually happens and who depends on what.